zio-http/src/main/scala/zhttp/http/Patch.scala

```diff
@@ -29,11 +30,13 @@ object Patch {
   final case class RemoveHeaders(headers: List[String]) extends Patch
   final case class SetStatus(status: Status)            extends Patch
   final case class Combine(left: Patch, right: Patch)   extends Patch
+  final case class UpdateHeaders(f: Headers => Headers) extends Patch
 
   def empty: Patch                                              = Empty
   def addHeader(headers: Headers): Patch                        = AddHeaders(headers)
   def addHeader(headers: Header): Patch                         = AddHeaders(Headers(headers))
   def addHeader(name: CharSequence, value: CharSequence): Patch = AddHeaders(Headers(name, value))
   def removeHeaders(headers: List[String]): Patch               = RemoveHeaders(headers)
   def setStatus(status: Status): Patch                          = SetStatus(status)
+  def updateHeaders(f: Headers => Headers): Patch               = UpdateHeaders(f)
```

Comment:
Can simply pull these changes in a separate PR. It will also get automatically documented in the release notes.

Reply:
Merged in main